examples/custom/build.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 set -e
 
+echo "SKIP_TEST:" ${SKIP_TEST}
```

Comment:
can we revert this change to the example project? It does demo this PR working, but not relevant to this example.